Misc/NEWS.d/next/Core and Builtins/2019-08-20-04-36-37.bpo-37751.CSFzUd.rst

```diff
@@ -0,0 +1 @@
+Fix ``normalizestring`` with hyphens and spaces converted to underscores and reuse ``_Py_normalize_encoding()`` in ``normalizestring``.
```

Comment:
normalizestring() is as private function, not visible to users. I propose to mention codecs.lookup() instead. I propose:

```
Fix :func:`codecs.lookup` to normalize the encoding name
the same way than :func:`encodings.normalize_encoding`,
except that :func:`codecs.lookup` also converts the name
to lower case.
```

My text is based on the _Py_normalize_encoding() comment:

```
/* Normalize an encoding name: similar to encodings.normalize_encoding(), but
   also convert to lowercase. Return 1 on success, or 0 on error (encoding is
   longer than lower_len-1). */
```

I compared _Py_normalize_encoding() and encodings.normalize_encoding(): the only difference is that _Py_normalize_encoding() converts also to lower case.

Subtle detail: _Py_normalize_encoding() only converts A-Z letters to a-z: it's different than str.lower() which implements the Unicode standard and considers the whole Unicode Character Set, not only ASCII. But I don't think that it's worth it to mention it here.

Reply:
Thank you for your patience and help. I have revised this issue.